map2loop/project.py

```diff
@@ -190,35 +199,38 @@ def __init__(
             self.map_data.set_filename(Datatype.DTM, dtm_filename)
         if fault_orientation_filename != "":
             self.map_data.set_filename(Datatype.FAULT_ORIENTATION, fault_orientation_filename)
+        
+        
         if config_filename != "":
             if clut_file_legacy:
                 print(
                     "DEPRECATION: Legacy files are deprecated and their use will be removed in v3.2"
                 )
             self.map_data.set_config_filename(config_filename, legacy_format=clut_file_legacy)
+        
         if config_dictionary != {}:
             self.map_data.config.update_from_dictionary(config_dictionary)
+        
         if clut_filename != "":
             self.map_data.set_colour_filename(clut_filename)
 
         # Load all data (both shape and raster)
         self.map_data.load_all_map_data()
 
         # If flag to save out data is check do so
+        tmp_path = pathlib.Path(tempfile.mkdtemp())
```

Comment:
If the user wants to save the data wouldn't they want to specify where? I agree with using a tempfile directory when the data is just tmp and not saved. But if the user is actually wanting to keep the data they need to be able to save the data. My other question is, if map2loop is run multiple times do you want the temp directory to stay the same or should it be a new temp directory per run? I actually think the temp directory made by tempfile should be create on module import not on create project to prevent creating too many temp files.